eppy/bunch_subclass.py

```diff
@@ -321,7 +321,13 @@ def __getattr__(self, name):
             try:
                 return self.fieldvalues[i]
             except IndexError:
-                return ""
+                # try to return the default value if defined in IDD.
+                if 'default' in self.getfieldidd(name).keys():
+                    _type = _parse_idd_type(self, name)
+                    default_ = next(iter(self.getfieldidd(name)['default']), None)
```

Comment:
I don't think this is python 2/3 compliant. `next(iterator)` is just python 2, while 3 uses `iterator.__next__()`.

Reply:
Looks like `next()` is defined in Python 3.7 https://docs.python.org/3.7/library/functions.html#next

Reply:
Quite right. I was thinking of `iterator.next()`